src/pages/office/components/candidate-description/candidate-description.ts

```diff
@@ -18,18 +24,27 @@ export class CandidateDescriptionComponent implements OnChanges {
 
   private formControl: FormControl;
 
+  constructor(private outcomeBehaviourProvider: OutcomeBehaviourMapProvider) { }
+
   ngOnChanges(): void {
     if (!this.formControl) {
-      this.formControl = new FormControl(null, [Validators.required]);
+      this.formControl = new FormControl(null);
       this.formGroup.addControl('candidateDescription', this.formControl);
     }
+    const visibilityType = this.outcomeBehaviourProvider.getVisibilityType(this.outcome, 'candidateDescription');
+
+    if (visibilityType === 'N') {
+      this.formGroup.get('candidateDescription').clearValidators();
+    } else {
+      this.formGroup.get('candidateDescription').setValidators([Validators.required]);
+    }
     this.formControl.patchValue(this.candidateDescription);
   }
 
   candidateDescriptionChanged(candidateDescription: string): void {
-    if (this.formControl.valid) {
-      this.candidateDescriptionChange.emit(candidateDescription);
-    }
+    //    if (this.formControl.valid) {
```

Comment:
I think validation is still required, can you enable these lines please?

Reply:
the removal of the form control valid is a fix for a bug raised by Andy.  We need to be able to save the office page with form validation errors so the examiner can complete later. (deferred test).

Have removed the comments